src/Rector/v9/v0/InjectAnnotationRector.php

```diff
@@ -73,11 +72,7 @@ public function refactor(Node $node): ?Node
         $injectMethods = [];
         $properties = $node->getProperties();
         foreach ($properties as $property) {
-            /** @var PhpDocInfo|null $propertyPhpDocInfo */
             $propertyPhpDocInfo = $this->phpDocInfoFactory->createFromNodeOrEmpty($property);
```

Comment:
This alwasy returns an `PhpDocInfo` object, even for empty doc

Reply:
Yeah. You do smarter analysis than me. This was some old code.